src/index.html

```diff
@@ -7,7 +8,58 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Stars</h1>
+    <section class="section">
+      <div class="stars stars--0">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
+      <div class="stars stars--1">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
+      <div class="stars stars--2">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
+      <div class="stars stars--3">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
+      <div class="stars stars--4">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
+      <div class="stars stars--5">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
+    </section>
```

Comment:
```suggestion
      </div>
    </section>
```
Don't add empty lines between parent and child elements